make_wininsts.bat

```diff
@@ -1,10 +1,10 @@
-SET PATH=C:\Python26;%PATH%
```

Comment:
would setting '* text=auto" in .gitattributes be preferred?  I have to
admit I've never really been that picky about line endings, or maybe just
used to working in a mixed Win/Unix environment.

On Thu, Oct 1, 2015 at 10:31 AM, Thomas A Caswell <notifications@github.com>
wrote:

> In make_wininsts.bat
> <https://github.com/pyepics/pyepics/pull/35#discussion_r40927207>:
>
> > @@ -1,10 +1,10 @@
> > -SET PATH=C:\Python26;%PATH%
>
> fair, sorry for being snippy.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyepics/pyepics/pull/35/files#r40927207>.
>



-- 
--Matt Newville <newville at cars.uchicago.edu> 630-252-0431